apps/e2e/modals-storybook-e2e/src/e2e/modal.component.cy.ts

```diff
@@ -0,0 +1,69 @@
+import { E2eVariations } from '@skyux-sdk/e2e-schematics';
+
+describe('modals-storybook', () => {
+  E2eVariations.forEachTheme((theme) => {
+    describe(`in ${theme} theme`, () => {
+      beforeEach(() =>
+        cy.visit(
+          `/iframe.html?globals=theme:${theme}&id=modalcomponent-modal--modal`
+        )
+      );
+      for (const x of [
```

Comment:
Use a variable name that's more descriptive than `x`. Perhaps `modalType`? Also, you should add a blank line above this block.